### PR TITLE
Cherry-pick #6302 to 5.6: fix packetbeat devices command

### PR DESCRIPTION
**Pull request by ph**
Cherry-pick of PR #6302 to 5.6 branch. Original message:

provide fpm deps

### CODE REVIEW

File: dev-tools/packer/docker/fpm-image/Dockerfile [hunk 1]
```diff
@@ -6,6 +6,6 @@ MAINTAINER Tudor Golubenco <tudor@elastic.co>
 RUN \
     apt-get update && \
     apt-get install -y --no-install-recommends \
-        build-essential ruby-dev rpm zip dos2unix libgmp3-dev
+        autoconf build-essential libffi-dev ruby-dev rpm zip dos2unix libgmp3-dev
 
 RUN gem install fpm
```
Comment (graphaelli) [inline dev-tools/packer/docker/fpm-image/Dockerfile]: consider pinning this to `-v 1.9.2` like is done on master

**Comment by ph** [comment]
@graphaelli updated.

File: dev-tools/packer/docker/fpm-image/Dockerfile [hunk 2]
```diff
@@ -1,11 +1,11 @@
-FROM ubuntu:16.04
+# FROM ubuntu:16.04
```
Comment (graphaelli) [inline dev-tools/packer/docker/fpm-image/Dockerfile]: did this `#` sneak in or is that intentional?

Comment (ph) [reply dev-tools/packer/docker/fpm-image/Dockerfile]: I really need coffee

Comment (ph) [reply dev-tools/packer/docker/fpm-image/Dockerfile]: Sorry for the waste of time.

Comment (graphaelli) [reply dev-tools/packer/docker/fpm-image/Dockerfile]: not at all, thanks for doing the backport

**Pull request merged**